src/js/components/Friends/SuggestedFriendList.jsx

```diff
@@ -37,9 +37,17 @@ export default class SuggestedFriendList extends Component {
           {this.state.suggestedFriendList.map((friend, index) => (
             <div key={friend.voter_we_vote_id}>
               <SuggestedFriendDisplayForList
```

Comment:
Hi @gautam-ergo, make sure to update the incoming props to camelCase within: src/js/components/Friends/SuggestedFriendDisplayForList.jsx